subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -22,10 +22,11 @@ config BT_FAST_PAIR_REQ_PAIRING
 	  procedure that is performed without the Bluetooth pairing operation.
 
 config BT_FAST_PAIR_PN
```

Comment:
In commit message should be `Personalized Name Extension` (it's `Personalized Extension` now)

Reply:
Also: `experimnetal` -> `experimental`

Reply:
fixed